frontend/package.json

```diff
@@ -19,8 +19,8 @@
     "keycloak-js": "12.0.4",
     "leaflet": "1.7.1",
     "lodash": "4.17.21",
-    "moment": "2.29.2",
-    "moment-timezone": "0.5.33",
+    "moment": "2.29.4",
+    "moment-timezone": "0.5.35",
```

Comment:
dont forget to update the package-lock file as well

Reply:
its there.